components/01-atoms/links/link.twig

```diff
@@ -0,0 +1,23 @@
+{#
+ # Available variables:
+ # - link_content - the content of the link (typically text)
+ # - link_url - the url this link should poing to
+ # - link_attributes - array of attribute,value pairs
+ #
+ # Available blocks:
+ # - link_content - used to replace the content of the link
+ #     Example: to insert the image component
+ #}
+
+{% set link_base_class = link_base_class|default('link') %}
+
+<a {{ bem(link_base_class, link_modifiers, link_blockname) }}
```

Comment:
Should we have a default `aria-label` attribute at this level or is there a way we can help enforce this where this component is used. Looking at Yale's accessibility rules it seems like `aria-label` or `aria-labelledby` or `aria-describedby` must be present. https://usability.yale.edu/web-accessibility/articles/links

Also, instead of using a for loop here can we take advantage of the `add_attributes` function somehow?

```
{% set link_attributes = link_attributes|default({
  'class':  bem(link_base_class, link_modifiers, link_blockname),
}) %}

<a add_attributes(link_attributes)>
```

Reply:
I don't think we need an aria-label right now. That would come in handy if it were on a card, and the link text was "Read more" then we used the card heading as the aria label so that it read out "Read more Awesome card heading" or something like that. A link by itself doesn't need an aria-label unless there is no text (like an icon/button) or if the text isn't helpful on its own (like "Read More")

And YES! I will use the add_attributes thing. I copied this from Compound as a starting point, and I didn't even consider updating it. On it now.

Reply:
@callinmullaney done